coinbase_tests.py

```diff
@@ -12,11 +14,17 @@
 
 
 
+root_path = os.path.dirname(os.path.realpath(__file__))
+if 'Darwin' in platform.system():
+    chromedriver_path = config.mac_chromedriver_path
+else: 
+    chromedriver_path = config.win_chromedriver_path
+
```

Comment:
I liked the way your Rocket repo let the user choose the path for their OS.  But I'm guessing this way will be more convenient. A Darwin will have to exist in the platform.system() for a mac OS to be present I take it.

Reply:
I'm not sure what you're talking about. The Rocket platform doesn't let the user "choose the path for their OS," if that means what I think it means.